### PR TITLE
Update whole-body-controllers in latest.releases.yaml to fix downstream usage of MomentumVelocityControl MATLAB functions

### PR DESCRIPTION
**Pull request by traversaro**
This fixes the problem reported in https://github.com/robotology/whole-body-controllers/issues/140 and fixed (on the whole-body-controllers side) on https://github.com/robotology/whole-body-controllers/pull/141 . 

Basically after https://github.com/robotology/whole-body-controllers/pull/141 the directory `<install_prefix>/mex/+wbc/simulink/MomentumVelocityControl` needs to be added to the MATLAB path. Once this change is released (in 2.5.6, I guess) this PR can be merged to make sure that the directory is added to the path once one sources the setup.sh script (if the robotology-superbuild is installed from sources) or once the conda environment is activated (if the `whole-body-controllers` conda package is installed).

### CODE REVIEW

**Comment by traversaro** [comment]
This PR is currently blocked until a new release of whole-body-controllers is done. fyi @gabrielenava @CarlottaSartore

**Comment by gabrielenava** [comment]
I've created a new release https://github.com/robotology/whole-body-controllers/releases/tag/v2.5.6 which include the modifications to the path done in this PR: https://github.com/robotology/whole-body-controllers/pull/141

In the near future I would like to finally reach a stable release for whole-body-controller, which will be the `v3.0`

**Comment by traversaro** [comment]
> I've created a new release https://github.com/robotology/whole-body-controllers/releases/tag/v2.5.6 which include the modifications to the path done in this PR: [robotology/whole-body-controllers#141](https://github.com/robotology/whole-body-controllers/pull/141)
> 
> In the near future I would like to finally reach a stable release for whole-body-controller, which will be the `v3.0`

Great, thanks! So this PR should be ready for review, I had already bumped the version in latest releases to v2.5.6.

File: cmake/template/setup.bat.in
Comment (gabrielenava) [inline cmake/template/setup.bat.in:51]: Just a small comment:  I didn't know there were examples installed in  `%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX%\mex\+wbc\examples`, is whole-body-controller installing them or another repo? In case it is another repo, we'll have to pay attention when refactor the installation path in WBC v3.0

**Review by gabrielenava — APPROVED** [review]
Looks good to me!

**Comment by traversaro** [comment]
The PR is ready for review @gabrielenava @CarlottaSartore @fabiodinatale @Andrea8Testa . Note that I totally imagine that you may not understand all the aspects of the PRs and the PR cannot be tested on its own, but the idea of the PRs in this context are:
* give the opportunity to people to see small changes in the infrastructure, and the freedom to ask questions
* catch clear typos

So feel free to approve even if you do not understand 100% of the PR. : )

**Review by CarlottaSartore — APPROVED** [review]

**Review by fabiodinatale — COMMENTED** [review]
Looks good to me! But I think that since it is already merged, I cannot approve it anymore.

**Pull request merged**
